app/controllers/authentication_controller.rb

```diff
@@ -51,7 +51,7 @@ def trigger_successful_publisher_sign_in_event(sign_in_type, publisher_oid = nil
       .with_type(:successful_publisher_sign_in_attempt)
       .with_request_details(request)
       .with_response_details(response)
-      .with_user(current_publisher)
+      .with_user(current_user)
```

Comment:
I don't know that these changes are ideal, e.g. after logging in a jobseeker we should never send anything other than a jobseeker along with the event.

Although we do currently log out different devise scopes when another logs in, I'm not sure whether that's the case for all three scopes, and that could change in future.

Might want to leave these as-were.